### DISCUSSION TITLE
Can you drag multiple items to sort at the same time?

### DISCUSSION THREAD

**Discussion by shaodahong** [clauderic/dnd-kit#581] [Q&A]
In the example, I only found a single drag sort, can drag multiple at the same time to sort?

**Comment by shaodahong** [comment]
Like this, `CMD+click` with drag https://react-beautiful-dnd.netlify.app/?path=/story/multi-drag--pattern

**Reply by microlabig** [reply]
Hello! 
Is there a solution how to drag and drop multiple items? 
I didn't find it on [examples](https://examples.dndkit.com/) page

**Reply by microlabig** [reply]
Previously i used `react-sortable-hoc`, but it's not supported right now.
As an example of use [https://github.com/clauderic/react-sortable-hoc/issues/318#issuecomment-473741137](https://github.com/clauderic/react-sortable-hoc/issues/318#issuecomment-473741137)
